src/frontend/packages/lib_components/src/data/stores/useVideo/index.tsx

```diff
@@ -55,4 +59,11 @@ export const useVideo = create<VideoState>((set, get) => ({
   removeResource: (video: Video) =>
     set(removeResource(get(), modelName.VIDEOS, video) as VideoStateResource),
   [modelName.VIDEOS]: {},
+  isWatchingVideo: false,
```

Comment:
you should also declare the `id3Video` property and set it to null. Once done you should change you declaration type accordingly